### ISSUE TITLE
[BUG] [Formatter] Bad indentation

### ISSUE THREAD

**Issue by tonial** [Riverside-Healthcare/djLint#432]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: Ubuntu 22.04
 - Python Version 3.10.6
 - djLint Version  1.19.2
 - template language: django

## Issue
Reformatting does work on 
```
<p>
    <span class="badge">New</span> You can now use <strong>this feature</strong>
</p>
```

## How To Reproduce

when using `djlint --reformat myfile.html`, the content becomes
```
<p>
    <span class="badge">New</span> You can now use <strong>this feature</strong>
    </p>
```

NB : 
- Without `You can now use` it works correctly
- With a `span` instead of `strong`, it also works correctly

I was able to get around this issue with a breakline. The following file doesn't cause the indentation error.
```
<p>
    <span class="badge">New</span>
    You can now use <strong>this feature</strong>
</p>
```

**Comment by tonial** [comment]
I tried to look into it : in both workaround cases, when processed in `indent_html()` the 2nd item (starting with `<span>` goes in to this condition  https://github.com/Riverside-Healthcare/djLint/blob/0d1f7222934a65aa71ecf259ca772bbebada66ae/src/djlint/formatter/indent.py#L68-L88  

The faulty file skip this condition and end in the `# if indent, move right` case

I don't know if the fix should be in `indent_html()` to allow this kind of lines, or if `expand_html()` should split the line after the `</span>`

**Comment by christopherpickering** [comment]
Thanks! Yep, you find the problem line, in indent_html. 
Right now that regex is not allowing any trialing text/html, but it should. I can update it, or if you already have something start feel free to send it over!

**Comment by tonial** [comment]
Hi, 
I tried do change the regex to allow it, but I'm afraid I'm not achieving much.
I'll guess I'll let you handle it :sweat_smile:

**Comment by christopherpickering** [comment]
Ok, thanks, no problem :)

**Comment by christopherpickering** [comment]
I loosened it up a bit, now it will check:
- line starts w/ a "single line tag"
- has some space or stuff
- ends with a "single line tag"
- maybe has some whitespace.
I think it fixes you problem and probably a few more..... Really need to get djlint changed to a parser instead of regex for stuff like this!

Thanks @tonial

**Comment by tonial** [comment]
That was fast !
Thanks for your work on this project !
